main/zoe/guide/proposal.md

```diff
@@ -1,41 +1,68 @@
-# The Structure of Proposals
+# The Structure of Offers
 
 <Zoe-Version/>
 
-## Making an offer
+## Making An offer
 
-In order to make an offer, you must provide two things: the payments
-that will be escrowed as part of the offer, and a statement of the
-conditions of the offer. This statement, which is approved by the user
-in their wallet, is known as the proposal.
+To make an offer, you use [`E(zoe).offer()`](https://github.com/Agoric/documentation/blob/Alpha-Zoe-Objects/main/zoe/api/zoe.md#ezoeofferinvitation-proposal-paymentkeywordrecord), which takes three arguments:
+- An `invitation` to participate in this contract instance.
+- A `proposal` stating your offer conditions.
+- The `payments` escrowed for the offer, each in association with a `proposal`-specified keyword.
 
-## Structure of the proposal
+## Invitations
+
+`Invitations` are a special case of ERTP `payments`. They are linked to a specific contract `instance`, and
+having one gives you the right to participate in that contract `instance`, for example, by making offers in it.
+
+There are two main ways for contract users to get an `invitation`:
+- If you create the contract `instance`, you get a special creator `invitation`.
+- Someone who holds the right objects has created an `invitation` for a contract `instance` and given it to
+  you in some way. This could've been by sending it to you, posting it on a public online location, etc. It
+  doesn't matter (nor does Zoe specify or have any requirements) how or why it got to you, only that you have it.
+
+## Proposals
 
 Proposals are objects with `give`, `want` and `exit` keys.
```

Comment:
I'm not sure we're consistent about this, but I think we should be saying that Proposals are records. Javascript probably doesn't distinguish, but the switch from records to objects was made because objects have methods and behavior. Proposals have fields, but no methods, to they're JavaScript Records.